app-modules/contact/src/Observers/ContactObserver.php

```diff
@@ -57,11 +57,11 @@ public function creating(Contact $contact): void
 
     public function created(): void
     {
-        Cache::tags('contacts')->flush();
+        Cache::tags('{{contacts}}')->flush();
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.